README.md

```diff
@@ -83,36 +80,44 @@ If you have something to add - anything from a typo through to a whole new featu
 You'll find that the [install script](https://github.com/pi-hole/pi-hole/blob/master/automated%20install/basic-install.sh) and the [debug script](https://github.com/pi-hole/pi-hole/blob/master/advanced/Scripts/piholeDebug.sh) have an abundance of comments, which will help you better understand how Pi-hole works. They're also a valuable resource to those who want to learn how to write scripts or code a program! We encourage anyone who likes to tinker to read through it, and submit a pull request for us to review.
 
 ### Presentations about Pi-hole
-Word-of-mouth continues to help our project grow immensely, and we'd like to help those who are going to be presenting Pi-hole at a conference, meetup or even a school project. If you'd like some free swag to hand out to your audience, [get in touch with us](https://pi-hole.net/2017/05/17/giving-a-presentation-on-pi-hole-contact-us-first-for-some-goodies-and-support/).
+Word-of-mouth continues to help our project grow immensely, and so we are helping make this easier for people.
+
+If you are going to be presenting Pi-hole at a conference, meetup or even a school project, [get in touch with us](https://pi-hole.net/2017/05/17/giving-a-presentation-on-pi-hole-contact-us-first-for-some-goodies-and-support/) so we can hook you up with free swag to hand out to your audience!
 
 -----
 
 ## Getting in touch with us
-- [Users Forum](https://discourse.pi-hole.net/)
-- [Feature requests](https://discourse.pi-hole.net/c/feature-requests?order=votes)
-- [FAQs](https://discourse.pi-hole.net/c/faqs)
-- [Wiki](https://github.com/pi-hole/pi-hole/wiki)
-- [/r/pihole on Reddit](https://www.reddit.com/r/pihole/)
-- [@The_Pi_Hole on Twitter](https://twitter.com/The_Pi_Hole)
-- [Pi-hole on YouTube](https://www.youtube.com/channel/UCT5kq9w0wSjogzJb81C9U0w)
-- [ThePiHole on Facebook](https://www.facebook.com/ThePiHole/)
-- [Chat on Gitter](https://gitter.im/pi-hole/pi-hole)
+While we are primarily reachable on our <a href="https://discourse.pi-hole.net/">Discourse User Forum</a>, we can also be found on a variety of social media outlets. **Please be sure to check the FAQ's** before starting a new discussion, as we do not have the spare time to reply to every request for assistance.
+
+<ul>
+  <li><a href="https://discourse.pi-hole.net/c/faqs">Frequently Asked Questions</a></li>
+  <li><a href="https://github.com/pi-hole/pi-hole/wiki">Pi-hole Wiki</a></li>
+  <li><a href="https://discourse.pi-hole.net/c/feature-requests?order=votes">Feature Requests</a></li>
+</ul>
```

Comment:
The list is ended here, resulting in uneven spacing.  Other than that, it looks good.
![screenshot 2017-11-19 22 46 35](https://user-images.githubusercontent.com/3843505/33002746-ba129ac0-cd7b-11e7-8dec-4a66d71bc7d2.png)

Reply:
Intentional - I've added a `<br>` to add some emphasis to it. 😄